Cargo.toml

```diff
@@ -9,6 +9,7 @@ base64 = "0.22"
 clap = { version = "4.5", features = ["derive"] }
 colored = "2.0"
 csv = "1.3"
+indicatif = "0.17"
```

Comment:
This is going to be a big change. We might want to break this out into a separate feature that works across all the tools

Reply:
On second though, just file an issue to port the rest of the tool to use this.